src/comitClient.ts

```diff
@@ -55,15 +54,20 @@ export class ComitClient {
 
   public async getOngoingSwaps(): Promise<Swap[]> {
     const swaps = await this.cnd.getSwaps();
-
     return swaps
-      .filter((swap: EmbeddedRepresentationSubEntity) => {
-        return (
-          swap.actions &&
-          !!swap.actions.find((action: Action) => {
-            return action.name === "fund" || action.name === "redeem";
-          })
-        );
+      .filter((swap: SwapSubEntity) => {
+        if (swap.properties && swap.properties.status === "IN_PROGRESS") {
+          if (swap.actions && swap.actions.length !== 0) {
+            return !!swap.actions.find((action: Action) => {
```

Comment:
I think there is some complexity in this if statements :D 

Can you simplify this a bit please, e.g. according the return early approach: 

```
if (!swap.properties) {
  return false;
}
if (swap.properties.status !== "IN_PROGRESS") {
  return false
}
...
```

This would improve the readability of this function.

Reply:
Resolving in #76